src/librustc/mir/interpret/error.rs

```diff
@@ -363,6 +363,8 @@ pub enum UndefinedBehaviorInfo {
     UbExperimental(String),
     /// Unreachable code was executed.
     Unreachable,
+    /// An enum discriminant was set to a value which was outside the range of valid values.
+    InvalidEnumDiscriminant,
```

Comment:
`InvalidDiscriminant` already exists as a variant ;)

Reply:
Oh, so it does. I should have looked harder :)

Does it matter that it's a `UnsupportedOpInfo` instead of an `UndefinedBehaviorInfo`?

Reply:
It's in the "not yet sorted" part of unsupported-op.

If you want yo can move it to the UB enum. ;)